admin/Default/enable_game_processing.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+if (!empty($_POST['game_id'])) {
+	// Enable the requested game
+	$game_id = $db->escapeNumber($_POST['game_id']);
```

Comment:
Can you put the escaping inline with the query - it just helps to make it obvious that the escaping has been done (I mean the ideal would be to use bind variables everywhere, but that'd be quite a huge change :( )